docs/parameters.md

```diff
@@ -0,0 +1,324 @@
+# Passing parameters to brokers
+
+Table of Contents
+- [Overview](#overview)
+- [Design](#design)
+  - [Basic example](#basic-example)
+  - [Passing sensitive data](#passing-sensitive-data)
+  - [Value types](#value-types)
+    - [String](#string)
+    - [JSON object](#json-object)
+    - [Other primitive types (int, bool, float)](#other-primitive-types-int-bool-float)
+    - [Arrays](#arrays)
+  - [Passing all parameters from a single source at once](#passing-all-parameters-from-a-single-source-at-once)
+  - [Conflict resolution](#conflict-resolution)
+
+## Overview
```

Comment:
@pmorie it's there already: [link](https://github.com/kubernetes-incubator/service-catalog/pull/1075/files/c600375969660af23e1952848c0e5851d2371669#diff-51b815dd3be8627b211d6accccdc7722R134)